docs/performance/benchmarks.rst

```diff
@@ -0,0 +1,4 @@
+Performance Tests - Benchmarks
+==============================
+
+Click `here <https://open-telemetry.github.io/opentelemetry-python-contrib/benchmarks/index.html>_` to view the latest performance benchmarks for packages in this repo.
```

Comment:
Need to wait for #252 to add this to the toc tree, otherwise we can just add the toc tree entry after #252 gets merged.